### PR TITLE
[Issue #2481] Add new GitHub export

### LINKED ISSUE #2481: Add new GitHub export script to `/analytics` codebase
### Summary

Add the new export script prototyped in the sandbox repo to the `/analytics` codebase so that it can be triggered using the CLI.

### Acceptance criteria

- [x] Running a CLI command similar to: `make delivery-data-export` will export and flatten the issue data from the GitHub project and write it to `tmp/delivery-data.json`

### PR DESCRIPTION
**Pull request by widal001**
## Summary

Adds new export command for the new GitHub issue structure.

Fixes #2481 

### Time to review: __10 mins__

- Adds `GitHubIssues` dataset to `datasets/issues.py`
- Enables tests that require a slack token to be run with the `--slack-token-set` pytest flag
- Creates a new `GitHubIssues` dataset that stores the content of the new delivery data export
- Refactors `integrations/github.py` to make it a sub-package `integrations/github/` to store the bash script and graphql queries
- Adds a new `export_delivery_data()` to `integrations/github/main.py`
- Replaces the old dataset in `export_json_to_database()` with `GitHubIssues`

### TODO

- [x] Refactor `integrations/github.py` to make it a sub-package `integrations/github/` to store the bash script and graphql queries
- [x] Add a new `export_delivery_data()` to `integrations/github/main.py`
- [x] Add a CLI entrypoint to export the data.
- [x] Replacing the old dataset in `export_json_to_database()`

1. Follow the instructions in `documentation/analytics/development.md` using Docker.
2. Run `make delivery-data-export POINTS_FIELD="Story Points"` to export GitHub data using the new format
3. Run `make gh-data-db-import` to load the exported data into the Postgres DB
4. You should see about 1065 rows inserted

Screenshot 2024-10-28 at 12 21 18 PM

### CODE REVIEW

**Comment by acouch** [comment]
Approach looks great so far 🚀

File: analytics/Makefile [hunk 1]
```diff
@@ -163,6 +164,17 @@ roadmap-data-export:
 	--project $(ROADMAP_PROJECT) \
 	--output-file $(ROADMAP_FILE)
 
+delivery-data-export:
```
Comment (coilysiren) [inline analytics/Makefile:167]: Do we want this running in AWS?

**Review by coilysiren — DISMISSED** [review]

**Review by DavidDudas-Intuitial — COMMENTED** [review]
LGTM

**Comment by widal001** [comment]
@coilysiren and @DavidDudas-Intuitial I fixed one bug with the Makefile ([fix(analytics): Extra space in Makefile variable](https://github.com/HHS/simpler-grants-gov/pull/2539/commits/9f8852a1ef74c4f20c5215e3da8fc92c2c280bdc)) that I didn't catch until I was working on the sprint burndown PR, would you all mind reviewing again?

**Review by acouch — APPROVED** [review]

**Review by DavidDudas-Intuitial — APPROVED** [review]

**Pull request merged**